bumble/core.py

```diff
@@ -27,11 +28,19 @@
 # -----------------------------------------------------------------------------
 # fmt: off
 
-BT_CENTRAL_ROLE    = 0
-BT_PERIPHERAL_ROLE = 1
+class Role(enum.IntEnum):
+    CENTRAL    = 0
+    PERIPHERAL = 1
 
-BT_BR_EDR_TRANSPORT = 0
-BT_LE_TRANSPORT     = 1
+BT_CENTRAL_ROLE    = Role.CENTRAL
+BT_PERIPHERAL_ROLE = Role.PERIPHERAL
+
+class Transport(enum.IntEnum):
```

Comment:
I'd recommend naming this `TransportType`, not to confuse with the `Transport` object from `bumble.transport.common`

Reply:
How about `BtTransport`?

Reply:
Mode ?

Reply:
Maybe `PhysicalTransport`, since that's the term used in the BT spec?